opentracing/src/main/java/opentracing/Tracer.java

```diff
@@ -56,4 +55,22 @@
    * and others consider the caller to be the parent and the receiver to be the child)
    */
   <T> Span join(String operationName, T carrier);
```

Comment:
I'm not sure. I guess the same example of forcing sampling via tags before the span is created applies here too. Will make it return a SpanBuilder.